man/appendNamedRegion-methods.Rd

```diff
@@ -24,8 +24,7 @@ Appends data to an existing named region.
   included as an additional column with the specified name. If \code{rownames = NULL} (default),
   no row names will be included.}
   \item{worksheetScope}{Optional character vector with worksheet name(s) to target a \code{name}
```

Comment:
Do the same rules mentioned as in the vignette hold here, i.e. `""` means global scope and `NA` means first matching name? If so, it would be good to specify that here.